### DISCUSSION TITLE
Is there a way to log or debug suggestion calculations to find the bottleneck?

### DISCUSSION THREAD

**Discussion by skoch13** [saghen/blink.cmp#573] [Q&A] [answered]
blink.cmp is being slow in the TS monorepo, while it works perfectly fine in another TS project. Is there a way to check what is consuming time to calculate?

**Comment by saghen** [comment, marked as answer]
You could add some `vim.print`s in the code for the LSP source but otherwise, you might want to wait for #480

**Reply by skoch13** [reply]
works for me, thank you
